### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#146] [General]
Hi Ladislav,

I became aware of your project in February. Now the year 2022 is coming to an end and QLog has developed into a very good logbook software during this time. It is a lot of fun to work with the software.

Therefore, at the end of the year I would like to thank you for your work and wish you much joy and success with the further development.

Merry Christmas to you and your family and all the best in the new year.

73, Wolfgang
DL2KI

**Comment by foldynl** [comment]
Hi Wolfgang,

thank you very.  it's also thanks to you that QLog can move forward.

I hope that next year QLog will improve again and will be beneficial to everyone who uses it. The main one will keep the development direction.

Marry Christmas and Happy New Year to you, your family and all OMs.

73, Ladislav
OK1MLG
